tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -211,6 +211,35 @@ public function userShouldGetANotificationWithMessage(string $user, string $subj
 		);
 	}
 
+	/**
+	 * @Then user :user should get last notification with subject :subject and message:
+	 *
+	 * @param string $user
+	 * @param string $subject
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function userShouldGetLastNotificationWithMessage(string $user, string $subject, TableNode $table):void {
```

Comment:
Can't we refactor the `userShouldGetANotificationWithMessage` step for this case?